src/components/CPU/BreakpointPanelContextMenu.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connectMenu, ContextMenu, MenuItem } from 'react-contextmenu';
+
+function BreakpointPanelContextMenu(props) {
+	const { id, trigger } = props;
+
+	return (
+		<ContextMenu id={id}>
+			{trigger?.breakpoint ? (<>
+				<MenuItem onClick={() => props.toggleBreakpoint(trigger.breakpoint, trigger.type)}>
+						Toggle Break
+				</MenuItem>
+				<MenuItem onClick={() => props.editBreakpoint(trigger.breakpoint, trigger.type)}>
+						Edit

```

Comment:
For Edit and Add New, we should have `...` at the end to indicate it will show a modal / further interaction.  I think the existing ones are actually three full stops, but should really be `&hellip;`.

-[Unknown]